apertium-crh.crh.lexc

```diff
@@ -8,65 +9,82 @@
 !!!                                                                         !!!
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
+! See List of symbols http://wiki.apertium.org/wiki/List_of_symbols
+
+! See http://wiki.apertium.org/wiki/Crimean_Tatar and http://wiki.apertium.org/wiki/Category:Turkic_languages
+
 Multichar_Symbols 
 
-%<n%>             ! Noun                        !
-%<np%>            ! Proper noun                 !
-%<adj%>           ! Adjective                   !
-%<prn%>           ! Pronoun                     !
-%<post%>          ! Postposition                !
-%<adv%>           ! Adverb                      !
-%<v%>             ! Verb                        !
-%<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
+%<n%>             ! Noun                        
+%<np%>            ! Proper noun                 
+%<adj%>           ! Adjective                   
+%<prn%>           ! Pronoun                     
+%<post%>          ! Postposition                
+%<adv%>           ! Adverb                      
+%<v%>             ! Verb                        
+%<cnjcoo%>        ! Coordinating conjunction    
+%<cnjsub%>        ! Sub-ordinating conjunction
+%<cnjadv%>        ! Adverbial conjunction 
 %<qst%>           ! Question word
-%<sym%>
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post Adverb
+%<det%>           ! Determiner
+%<num%>           ! Numeral
 %<vaux%>          ! Auxiliary verb
 %<abbr%>          ! Abbreviation
-%<emph%> 
-%<ij%>
-%<nom%>           ! Nominative                  !
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
```

Comment:
Why was this removed?

Reply:
This isnt removed they are available under cases heading

Reply:
What line number?

Reply:
loc-60 
abl-59
ins-61

Reply:
sim-78

Reply:
oh okay i will change that